docs/observation/payload.md

```diff
@@ -1,34 +1,89 @@
 # Format of observation payload
 
-Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the respective listener (e.g. `onedrive-change-listener` for OneDrive), they get converted to a portable format before sent to the next stage. The format has the following properties:
+
+| Name | Type | Description |
+|------|------|-------------|
+| `type` | _string_ | provider type, possible values are `onedrive`, `github` or `google`. |
+| `owner` | _string_ | GitHub owner |
+| `repo` | _string_ | GitHub repository name |
+| `ref` | _string_ | GitHub reference or branch name |
+| `changes` | _array_ | [changes](#changes) observed, see below |
+| `mountpoint` | _object_ | [mountpoint](#mountpoint) affected, see below |
+| `provider` | _object_ | provider specific information |
+
+An example payload for OneDrive might look as follows:
 
 ```
 {
   "type": "onedrive",
-  "provider" {
-    "driveId": "..."
-  },
   "owner": "me",
   "repo": "myrepo",
   "ref": "master",
-  "mountpoint": {
-    "path": "/ms/",
-    "root": "/myshare"
-  },
   "changes": [
     ...
   ],
+  "mountpoint": {
+    ...
+  },
+  "provider": {
+    "driveId": "b!-RIj2DuyvEyV1T4NlOaMHk8XkS_I8MdFlUCq1BlcjgmhRfAj3-Z8RY2VpuvV_tpd"
```

Comment:
Note: that `driveId` is the sample account's root drive available in Microsoft's Graph Explorer